<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DELL" name="com.actiTime.File.TestngFeatures" tests="11" failures="0" timestamp="2020-12-07T21:17:58 IST" time="0.171" errors="1">
  <testcase name="add" time="0.002" classname="com.actiTime.File.TestngFeatures"/>
  <system-out/>
  <testcase name="infiniteloop" time="0.116" classname="com.actiTime.File.TestngFeatures">
    <error type="org.testng.internal.thread.ThreadTimeoutException" message="Method com.actiTime.File.TestngFeatures.infiniteloop() didn&#039;t finish within the time-out 2">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method com.actiTime.File.TestngFeatures.infiniteloop() didn't finish within the time-out 2
at org.testng.internal.MethodInvocationHelper.invokeWithTimeoutWithNewExecutor(MethodInvocationHelper.java:371)
at org.testng.internal.MethodInvocationHelper.invokeWithTimeout(MethodInvocationHelper.java:282)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:605)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- infiniteloop -->
  <system-out/>
  <testcase name="add" time="0.004" classname="com.actiTime.File.TestngFeatures"/>
  <system-out/>
  <testcase name="add" time="0.020" classname="com.actiTime.File.TestngFeatures"/>
  <system-out/>
  <testcase name="add" time="0.005" classname="com.actiTime.File.TestngFeatures"/>
  <system-out/>
  <testcase name="add" time="0.002" classname="com.actiTime.File.TestngFeatures"/>
  <system-out/>
  <testcase name="add" time="0.004" classname="com.actiTime.File.TestngFeatures"/>
  <system-out/>
  <testcase name="add" time="0.003" classname="com.actiTime.File.TestngFeatures"/>
  <system-out/>
  <testcase name="add" time="0.003" classname="com.actiTime.File.TestngFeatures"/>
  <system-out/>
  <testcase name="add" time="0.009" classname="com.actiTime.File.TestngFeatures"/>
  <system-out/>
  <testcase name="add" time="0.003" classname="com.actiTime.File.TestngFeatures"/>
  <system-out/>
</testsuite> <!-- com.actiTime.File.TestngFeatures -->
